app/assets/stylesheets/mortgage_calculator/components/_stamp-duty-calc.scss

```diff
@@ -0,0 +1,134 @@
+.stamp-duty__step-one {
+  margin-top: 0;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    margin-top: $baseline-unit*5;
+  }
+}
+
+.stamp-duty {
+  border: solid $color-grey-light 1px;
+  overflow: hidden;
+  padding: 0 $baseline-unit*4;
+  max-width: none;
+  margin-top: $baseline-unit;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    margin-top: $baseline-unit*8;
+  }
+}
+
+.stamp-duty__heading {
+  @include body(24,36);
+  margin-top: $baseline-unit*4;
+  margin-bottom: $baseline-unit;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    @include body(40,45);
+  }
+}
+
+.stamp-duty__subheading {
+  @include body(16,24);
+  font-weight: 300;
+  margin-bottom: 0;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    @include body(21,24);
+  }
+}
+
+.stamp-duty__form {
+  margin-top: 0;
+  overflow: hidden;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    margin: 0 0 $baseline-unit*3;
+  }
+}
+
+.stamp-duty__input-description {
+  display: block;
+  @include body(14,24);
+  color: $color-true-black;
+  font-weight: 600;
+  padding: $baseline-unit*2 0;
+  margin: 0;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    float: left;
+    display: inline-block;
+    @include body(21,24);
+  }
+}
+
+input.stamp-duty__input {
```

Comment:
It's annoying that we need the `input` selector here - I didn't know that attribute selectors carried the same specificity as class selectors. I don't think there's a better way to do this though.